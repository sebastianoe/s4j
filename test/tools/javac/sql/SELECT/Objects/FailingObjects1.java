/*
 * @test  (c) SO
 * @summary Fail, since table doesn't exist
 * @compile businessobjects/Product.java
 * @compile businessobjects/Storage.java
 * @compile/fail/ref=FailingObjects1.out FailingObjects1.java -sqlrepo /Users/sebastian/ma/jdk7u-langtools/test/tools/javac/sql/repo.json
 */

import businessobjects.Product;
import businessobjects.Storage;

public class FailingObjects1 {
    public static void main(String[] args) {
    	Product p = SQL[SELECT * FROM anything];
    }    
}