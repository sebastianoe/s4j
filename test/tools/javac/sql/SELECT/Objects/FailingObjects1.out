error: The table anything does not exist in the provided DB schema information
1 error